src/main.rs

```diff
@@ -28,11 +28,11 @@ lazy_static::lazy_static! {
 
 fn parse_username(text: &str) -> Result<String, &'static str> {
     if text.is_empty() {
-        return Err("Username shouldn't be empty");
+        return Err("/send username cannot be empty. To send emojis to someone follow the format: `/send QUANTITY EMOJI USERNAME`");
     }
 
     if text.chars().nth(0).unwrap() != '@' {
-        return Err("Username should start with `@`");
+        return Err("Username should start with `@`. You shouldn't see this error, so if you do please contact a game admin at https://t.me/joinchat/iE9YbriRxldlMWZh");
```

Comment:
Why the user shouldn't see this error? 🤔

Reply:
eu confundi achei que os erros aqui fossem de outro contexto e arrumei só o primeiro! vou consertar

Reply:
@otaviopace eu acho que em termos de UX seria melhor aqui aceitar tanto com quanto sem @, ai esse erro nem existiria, o que acha?

Reply:
o unico erro que existiria ai é o de usuario nao encontrado mesmo :)

Reply:
Faremos um PR depois pra isso 😊